src/test/java/usecase/deleteapplication/DeleteApplicationOutputDataTest.java

```diff
@@ -0,0 +1,131 @@
+package usecase.deleteapplication;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+/**
+ * Unit tests for the DeleteApplicationOutputData class and related functionality.
+ */
+public class DeleteApplicationOutputDataTest {
+
+    /**
+     * Tests that the application is deleted successfully.
+     */
+    @Test
+    void deleteApplicationSuccessfully() {
+        int senderUserId = 1;
+        int projectId = 1;
+
+        DeleteApplicationController deleteApplicationController = new DeleteApplicationController(mock(DeleteApplicationInteractor.class));
+        deleteApplicationController.deleteApplication(senderUserId, projectId);
+
+        // Verify the application was deleted successfully
+        // This would typically involve checking some state or output
+    }
+
+    /**
+     * Tests that an exception is thrown when an invalid user ID is provided.
+     */
+    @Test
+    void deleteApplicationWithInvalidUserIdThrowsException() {
+        int invalidUserId = -1;
+        int projectId = 1;
+
+        DeleteApplicationController deleteApplicationController = new DeleteApplicationController(mock(DeleteApplicationInteractor.class));
+
+        assertThrows(IllegalArgumentException.class, () -> deleteApplicationController.deleteApplication(invalidUserId, projectId));
```

Comment:
deleteApplication does not throw exceptions (neither do the method it uses)

Reply:
@Raqkkkkk 
Thank you for catching that! It is due to a broken test case I introduced earlier (somehow there was a throw exception I accidently added in a class of my test code and the it was referring to that). 

Anyway I have fixed the issue and tested with 100% coverage of that package.